api/koko_api.go

```diff
@@ -620,9 +628,11 @@ func MakeVeth(veth1 VEth, veth2 VEth) error {
 
 	if err = veth1.SetVethLink(link1); err != nil {
 		netlink.LinkDel(link1)
+		netlink.LinkDel(link2)
 		return err
 	}
 	if err = veth2.SetVethLink(link2); err != nil {
+		netlink.LinkDel(link1)
```

Comment:
We don't need this because veth must be exists with pairs of veth. So if one is removed then another must be removed. Is there any clear corner cases which one veth is still remained?